src/DependencyInjection/Compiler/PurgeTypeSettingsPass.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace EzSystems\PlatformHttpCacheBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class PurgeTypeSettingsPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container)
+    {
+        $purgeType = $container->getParameter('purge_type');
```

Comment:
we should not use the parameter here... that could be renamed. we should use the ezpublish.htt_cache.purge_type settings

Reply:
Hmm.. Not sure that is possible in a compiler pass. You know how?

Reply:
Not by default, I think, no way :( But for sure we should do that on `purge_type`.

You will have to set the value as a parameter here:
https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Bundle/EzPublishCoreBundle/DependencyInjection/EzPublishCoreExtension.php#L392
```php
$container->setParameter("ezpublish.http_cache.purge_type", $config['http_cache']['purge_type']);
```

And then refer to it 
```php
$purgeType = $container->getParameter('ezpublish.http_cache.purge_type');
```

Then comes the SiteAccess Awareness, but I think that is global then a parameter would be fine.

@andrerom  @bdunogier  another idea?

Reply:
Right, fixed as suggested